### PR TITLE
Feat/694 Perpetual second pair preperation

### PR DESCRIPTION
**Pull request by ShynRou**
Resolves https://taiga.sovryn.app/project/sovryn-light/task/694

### CODE REVIEW

File: src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx [hunk 1]
```diff
@@ -164,6 +177,26 @@ export const PerpetualQueriesContextProvider: React.FC<PerpetualQueriesContextPr
...
+            .call(
+              Chain.BSC,
+              getContract('PERPETUALS_token').address,
+              marginTokenAbi,
+              'balanceOf',
+              [account],
+            )
+            .then(result => result && setAvailableBalance(String(result)))
+            .catch(console.error),
+        500,
```
Comment (tiltom) [inline src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx:194]: Why do you have `500 ms` here? We use `1000 ms` (`THROTTLE_DELAY`) everywhere else.

File: src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx [hunk 2]
```diff
@@ -164,6 +177,26 @@ export const PerpetualQueriesContextProvider: React.FC<PerpetualQueriesContextPr
...
+        { leading: false, trailing: true, maxWait: 500 },
```
Comment (tiltom) [inline src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx:195]: `leading: false` and `traling: true` are the default values (according to the [documentation](https://lodash.com/docs/4.17.15#debounce)) so there is no need to specify them.

File: src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx [hunk 3]
```diff
@@ -128,29 +143,27 @@ export const PerpetualQueriesContextProvider: React.FC<PerpetualQueriesContextPr
...
-    debounce(() => refetchTraderState(), THROTTLE_DELAY, {
-      leading: true,
-      trailing: true,
-      maxWait: THROTTLE_DELAY,
-    }),
+  const refetchUser = useMemo(
+    () =>
+      debounce(() => refetchTraderState(), THROTTLE_DELAY, {
+        leading: true,
+        trailing: true,
```
Comment (tiltom) [inline src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx:164]: `true` is the default value of `trailing` so there is no need to specify it.

**Review by tiltom — CHANGES_REQUESTED** [review]

File: src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx (hunk 1 above)
Comment (ShynRou) [reply src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx:194]: Idk sounded like a reasonable value :D

**Pull request merged**